src/app/auth/page.tsx

```diff
@@ -0,0 +1,19 @@
+import AuthForm from '@/components/Pages/Auth/AuthForm'
+import ImageCard from '@/components/Pages/Auth/ImageCard'
+import Image from 'next/image'
+
+export default function Auth() {
+  return (
+    <section className="h-screen w-screen flex items-center justify-center bg-gradient-blue-white">
+      <div className="card p-10 flex gap-[3.75rem] bg-white">
+        <ImageCard className="-my-[4.375rem] flex-shrink-0">
+          <Image alt="Golub" height={161} src="/images/golub.png" width={196} />
+        </ImageCard>
+        <div>
+          <h1 className="title-lg">Authentication</h1>
+          <AuthForm />
```

Comment:
Very subjectively, but I would name it `FormAuth` instead to make a common prefix at the beginning. Then all forms will start with `Form*` prefix. 

P.S. the same as we have `IconCheck` now

Reply:
OK, that makes sense